src/containers/Welcome/Step2/Step2.tsx

```diff
@@ -73,6 +76,8 @@ const Step2 = (p: Wizard.StepProps) => {
 
   const { t } = useTranslation();
 
+  if (country === 'Japan' && p.nextStep) history.push(p.nextStep);
```

Comment:
On render? 
I prefer you to use a effect for this and return something empty on body 🤔
Anyway, remember that we have some step alternatives that we can include and use that for calculate nextStep. So can jump to Step 3 from Step 1 if you use that logic (or the previous step for this)
I prefer this last option, because it will be a clean history for the user. (You can keep the push if you want for those cases where the user put the url by hand)